guide/contribute/propose.md

```diff
@@ -0,0 +1,173 @@
+
+{% include youtube.html id="0vMBdtOnHJM" %}
+
+### Prerequisites
+
+While advanced users can interact with GitHub through a command line, this guide focuses on showing how to work with Git locally, by using the following software:
+
+1. Create a [GitHub account](https://github.com/)
+2. [Download and install GitHub Desktop](https://desktop.github.com/) (macOS and Windows) / [Linux](https://github.com/shiftkey/desktop)
+3. Download and install text editor ([Sublime Text](https://www.sublimetext.com/), [Atom](https://atom.io/), etc)
+
+### Identifying what to fix
+
+If you're looking to help, but not sure where to begin:
+
+- Check issues labeled as [Good first issue](https://github.com/BitcoinDesign/Guide/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
+- Check issues labeled as [Design](https://github.com/BitcoinDesign/Guide/issues?q=is%3Aissue+is%3Aopen+label%3Adesign)
+- Consult the community [on Slack]({{ site.slack_invite_url }}) or start a [GitHub discussion](https://github.com/BitcoinDesign/Guide/discussions/new) with your proposal
+- Browse [bitcoin.design](https://bitcoin.design/guide) test it and try to identify issues yourself
+
+For more challenging pieces of work, a new page, or a new chapter, it's a good idea to talk to the community and seek consensus before you begin working on it. This helps avoid work duplication and makes sure your efforts are aligned with the community-determined goals.
+
+### Getting started
+
+Like any novelty, the process may seem intimidating at first, but the best way to understand this is to try it out in practice.
+
+{% include youtube.html id="DxrLZtABG0E" %}
+
+1. [Create a fork (clone)](#forking-design-guide-repository)
+2. [Create a branch](#creating-a-branch)
+3. [Manage files](#managing-files)
+4. [Apply changes](#applying-changes)
+5. [Create a pull request](#creating-a-pull-request)
+6. [Applying additional changes](#applying-additional-changes)
+
+#### Forking Design Guide repository
+
+The first step is to fork the Bitcoin Design Guide repository. Forking a repository means that you're creating an exact clone (copy) of the Bitcoin Design Guide locally on your computer and in a cloud on your GitHub.com account.
+
+To fork a repository[^1]:
+
+- Sign in to GitHub desktop
+- File > `Clone repository`
+- Switch to the URL tab
+- Paste `https://github.com/BitcoinDesign/Guide/` into the URL field.
+- Click `Clone`
+
+{% include picture.html
+   image = "/assets/images/guide/contribute/propose/clone-repository.png"
+   retina = "/assets/images/guide/contribute/propose/clone-repository@2x.png"
+   alt-text = "Cloning a repository"
+   width = 400
+   height = 400
+%}
+
+#### Creating a branch
+
+A branch[^2] allows you to have a _contained area_ where you can make any changes, without going out of sync with the cloned version.
```

Comment:
What does [^2] do? Seems to link to Git docs, but how? Never seen that before lol

Reply:
Hey it's a footnote that gets added at the bottom of the article :smile_cat: 
![Screenshot from 2021-01-21 17-45-41](https://user-images.githubusercontent.com/36959754/105382652-98ef8c00-5c10-11eb-9450-5b9a39f35251.png)